lib/index.js

```diff
@@ -56,6 +56,7 @@ Plotly.register([
 
     require('./sankey'),
     require('./indicator'),
+    require('./image'),
```

Comment:
Can you move this `require` statement up in this block:

https://github.com/plotly/plotly.js/blob/e86c95b4b2abe646d7ab4e311fcd40cc61f1eaea/lib/index.js#L15-L25

with all the other cartesian traces.

Reply:
Done in 73c0740f53fc82a70196d1993ba2824db2be85a3